### ISSUE TITLE
Add option block removal of existing groups

### ISSUE THREAD

**Issue by ltfarm** [yllen/behaviors#96]
Hello,

Just a feedback, so we can try to improve the usability of the system.

Context (premise): every ticket must have a group and technician.

Problem: When groups move their tickets to other groups or even add a technician to the ticket, during this process it becomes possible to exclude the ticket group or other actors. In this sense, it would be interesting when adding a new group to not be able to remove the existing one. Because, when adding a new technician or group, these are already replaced (if the group/single technician option). Because, in a practical environment, it creates problems, as the technician removes the group and the queue no longer has a view of the ticket.

Solution
In other words, generally speaking, it would be nice to have an option to block the removal of existing groups.

Greetings

**Comment by yllen** [comment]
You can't block the removal existing group if you had activate the parameter "single group", because you won't be able to add the new group.
I can't have 2 divergent parameters in the configuration.
I don't know how to do that

**Comment by leandrovc** [comment]
If the group input element was changed to a single selection type when single group was activated, that would do it, but it would also be necessary to split the group and technician into two separate fields.
